NEWS.md

```diff
@@ -1,5 +1,7 @@
 # readr (development version)
 
+* readr is now licensed as MIT (#1189, #1190).
```

Comment:
@jimhester might suggest a better home for this bullet but otherwise looks good.